### PR TITLE
CMD (curl) format example

### PR DESCRIPTION
**Pull request by vaclavdobsicek**

### CODE REVIEW

File: docs/extensions/netsuite/considerations.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ Request payload example:
 }
 ```
 
+### CURL version example
```
Comment (mrtnzlml) [inline docs/extensions/netsuite/considerations.md:31]: The title might potentially be a bit unnecessary. Consider following up with a simple sentence describing that "you can execute this change via the following command…"

File: docs/extensions/netsuite/considerations.md [hunk 2]
```diff
@@ -28,6 +28,14 @@ Request payload example:
...
+```bash
+curl --location --request PATCH 'https://[EAMPLE].rossum.app/api/v1/hooks/[HOOK_ID]' \
+--header 'Authorization: token [API_TOKEN]' \
```
Comment (mrtnzlml) [inline docs/extensions/netsuite/considerations.md:34]: Shouldn't this actually be:

```suggestion
--header 'Authorization: Bearer [API_TOKEN]' \
```

🤔

File: docs/extensions/netsuite/considerations.md [hunk 3]
```diff
@@ -28,6 +28,14 @@ Request payload example:
...
+--header 'Content-Type: application/json' \
+--data '{"token_lifetime_s": 120}'
```
Comment (mrtnzlml) [inline docs/extensions/netsuite/considerations.md:36]: Shouldn't this be:

```suggestion
--data '{"timeout_s": 120}'
```

🤔

**Review by mrtnzlml — CHANGES_REQUESTED** [review]
@vaclavdobsicek I am not sure about some of the proposed changes. Could you please check my comments? Especially the ones related to the actual cURL command.

File: docs/extensions/netsuite/considerations.md (hunk 3 above)
Comment (vaclavdobsicek) [reply docs/extensions/netsuite/considerations.md:36]: You're right, the correct parameter for timeout is "timeout_s"

File: docs/extensions/netsuite/considerations.md [hunk 4]
```diff
@@ -28,6 +28,14 @@ Request payload example:
 }
 ```
 
+### CURL version example
+```bash
+curl --location --request PATCH 'https://[EAMPLE].rossum.app/api/v1/hooks/[HOOK_ID]' \
+--header 'Authorization: Bearer [API_TOKEN]' \
+--header 'Content-Type: application/json' \
+--data '{"timeout_s": 120}'
```
Comment (mrtnzlml) [inline docs/extensions/netsuite/considerations.md:36]: @vaclavdobsicek One more change, please. Users cannot set more than 60 seconds:

```suggestion
--data '{"timeout_s": 60}'
```

**Review by mrtnzlml — CHANGES_REQUESTED** [review]
Sorry, my mistake. There is one more mistake that needs to be fixed before merging. 🙈

**Review by mrtnzlml — APPROVED** [review]

**Pull request merged**